crit-go/gocrit/cmdhandler.go

```diff
@@ -0,0 +1,115 @@
+package gocrit
+
+import (
+	"crit-go/images"
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+func Check(e error) {
+	if e != nil {
+		fmt.Println(e)
+		os.Exit(1)
+	}
+}
+
+func checkfile(e error, f *os.File) {
+	if e != nil {
+		f.Close()
+		fmt.Println(e)
+		os.Exit(1)
+	}
+}
+
+func Decode(inloc string, outloc string, pretty bool, nopl bool) {
+	img := images.Load(inf(inloc), pretty, nopl)
+	ouf := outf(outloc)
+	if pretty == true {
+		encoder := json.NewEncoder(ouf)
+		encoder.SetIndent("", "    ")
+		err := encoder.Encode(&img)
+		checkfile(err, ouf)
+		ouf.Close()
+	} else {
+		encoder := json.NewEncoder(ouf)
+		err := encoder.Encode(&img)
+		checkfile(err, ouf)
+		ouf.Close()
+	}
+}
+
+func Encode(inloc string, outloc string) {
+	images.Dump(inf(inloc), outf(outloc))
+}
+
+func Info(inloc string) {
+	fmt.Println("test placeholder - Info called ")
+}
+
+func Explore(dir string, what string) {
+	switch what {
+	case "ps":
+		explore_ps(dir)
+	case "fds":
+		explore_fds(dir)
+	case "mss":
+		explore_mems(dir)
+	case "rss":
+		explore_rss(dir)
+	}
+}
+
+func Show(inloc string) {
```

Comment:
what does this function do?

Reply:
this is the show function basically decodes with nopl option. similar to the python version.